docs/src/main/asciidoc/datasource.adoc

```diff
@@ -511,154 +504,155 @@ public class TestResources {
-Going into each of those options is beyond the scope of this document,
-but the following section gives an overview of each database URL and a link to the official documentation.
+The following section gives an overview of each database URL and a link to the official documentation.
 
-=== DB2
+==== DB2
 
 `jdbc:db2://<serverName>[:<portNumber>]/<databaseName>[:<key1>=<value>;[<key2>=<value2>;]]`
 
 Example:: `jdbc:db2://localhost:50000/MYDB:user=dbadm;password=dbadm;`
 
 See the https://www.ibm.com/support/knowledgecenter/SSEPGG_11.5.0/com.ibm.db2.luw.apdv.java.doc/src/tpc/imjcc_r0052342.html[official documentation] for more detail on URL syntax and additional supported options.
 
-=== Derby
+==== Derby
 
 `jdbc:derby:[//serverName[:portNumber]/][memory:]databaseName[;property=value[;property=value]]`
 
 Example:: `jdbc:derby://localhost:1527/myDB`, `jdbc:derby:memory:myDB;create=true`
 
 Derby is an embedded database.
-It can run as a server, based on a file, or live completely in memory.
+It can run as a server, based on a file, or run completely in memory.
 All of these options are available as listed above.
-You can find more information at the https://db.apache.org/derby/docs/10.8/devguide/cdevdvlp17453.html#cdevdvlp17453[official documentation].
+You can find more information in the link:https://db.apache.org/derby/docs/10.8/devguide/cdevdvlp17453.html#cdevdvlp17453[official documentation].
 
-=== H2
+==== H2
 
 `jdbc:h2:{ {.|mem:}[name] | [file:]fileName | {tcp|ssl}:[//]server[:port][,server2[:port]]/name }[;key=value...]`
 
 Example:: `jdbc:h2:tcp://localhost/~/test`, `jdbc:h2:mem:myDB`
 
 H2 is an embedded database.
-It can run as a server, based on a file, or live completely in memory.
+It can run as a server, based on a file, or run completely in memory.
 All of these options are available as listed above.
-You can find more information at the https://h2database.com/html/features.html?highlight=url&search=url#database_url[official documentation].
 
-=== MariaDB
+You can find more information in the link:https://h2database.com/html/features.html?highlight=url&search=url#database_url[official documentation].
+
+==== MariaDB
 
 `jdbc:mariadb:[replication:|failover:|sequential:|aurora:]//<hostDescription>[,<hostDescription>...]/[database][?<key1>=<value1>[&<key2>=<value2>]]`
-hostDescription:: `<host>[:<portnumber>]  or address=(host=<host>)[(port=<portnumber>)][(type=(master|slave))]`
+hostDescription:: `<host>[:<portnumber>] or address=(host=<host>)[(port=<portnumber>)][(type=(master|slave))]`
 
 Example:: `jdbc:mariadb://localhost:3306/test`
 
-You can find more information about this feature and others detailed in the https://mariadb.com/kb/en/library/about-mariadb-connector-j/[official documentation].
+More information about this feature and others is detailed in the link:https://mariadb.com/kb/en/library/about-mariadb-connector-j/[official documentation].
```

Comment:
Any reason for using different phrasing for the "more information" sentence for H2 and MariaDB?

And FWIW, the next sections use yet another phrasing.

Reply:
I just used a minimalist version of the original sentence. For this content to be ever used as corporate docs (downstream version), some actions, such as the elimination of You can, you should, please, might, and so on....should be replaced with something that is OK with the IBM style guide, and yet not breaking the original intention. Yes, this is not the funny part at all :)

Reply:
Okay but you kept "You can find more information" for H2, that's why I found this change odd.

And I found it odd that some changes use "More information about this feature and others is detailed" and some use "More details can be found". I thought you'd want to use consistent phrasing, while you are at it.

Reply:
Exactly. My appologies, I had this changes in a commit but I havent pushed it. I just applied the commit and rebased. The suggestion  I resolved are now applied.